staff/sergio-rodriguez/playground/store/main.js

```diff
@@ -0,0 +1,130 @@
+/* ME HICE UN LÍO CON LOS CARID E HICE UN MAL COPY/PASTE DESDE SUBLIME!! HAY QUE REMIRAR
+EL UPDATECAR. MUCHO OJO!!! */
+
+var products = []
+
+// Manual population/populate (manual fill data trought JS)
+
+var cinquecento = {
+    id: "20250315-1601",
+    brand: "Fiat",
+    model: "500",
+    color: "Pink",
+    year: 2019,
+    kilometers: 145000
+}
+
+products[0] = cinquecento
+
+var clio = {
+    id: "20250315-1601",
+    brand: "Renault",
+    model: "500",
+    color: "red",
+    year: 2015,
+    kilometers: 225000
+}
+
+products[1] = clio
+
+var c3 = {
+    id: "20250315-1605",
+    brand: "Citröen",
+    model: "C3",
+    color: "Black",
+    year: 2010,
+    kilometers: 310000
+}
+
+products[2] = c3
+
+console.table(products);
+
+function generateRandomId() {
+    // Get current date
+    const now = new Date();
```

Comment:
sergio, do not use `const` yet. we are using `var`. we will use `const` in the following days, but not in this challenge.